### DISCUSSION TITLE
Macro for writing text

### DISCUSSION THREAD

**Discussion by ramiror** [brookhong/Surfingkeys#1460] [Q&A]
Is there a way to map a key to writing a text into a text input? I'd love to have a shortcut to write my complete email address in a text input.

**Comment by kopach** [comment]
I use https://espanso.org/ for such text expansion purposes. But still I personally would love to have "vim-like" macros in the extension itself. So, e.g. I could record a complex sequence of extension's actions and reply them with easy shortcut
